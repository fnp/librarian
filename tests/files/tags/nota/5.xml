<naglowek_podrozdzial>II</naglowek_podrozdzial>

<nota><akap>Moje wypadki • Ognisko w wąwozie • Uczta • Opowiadania skwatera • Nocleg • Głosy nocne
• Niespokojne sny • Ranek • Polowanie z zasadzki • Pierwszy jeleń</akap></nota>
