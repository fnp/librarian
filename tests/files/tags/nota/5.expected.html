<div id="book-text">
<h4 id="s1">II</h4>
<div class="note"><p class="wl paragraph" id="i1">Moje wypadki • Ognisko w wąwozie • Uczta • Opowiadania skwatera • Nocleg • Głosy nocne
• Niespokojne sny • Ranek • Polowanie z zasadzki • Pierwszy jeleń</p></div>
</div>
